benchmarks/apps/trove/trove.py

```diff
@@ -0,0 +1,143 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+from benchmarks.modules.utils import SpackTest
+
+
+class Trove(SpackTest):
+
+    descr = 'Base class for Trove'
+    valid_systems = ['*']
+    valid_prog_environs = ['*']
```

Comment:
```suggestion
    valid_prog_environs = ['default']
```
Also related to comment on sphng :grinning: